runtime/themes/noctis.toml

```diff
@@ -0,0 +1,199 @@
+
+# UI ==============================
+# For styling helix itself.
+
+'ui.background' = { bg = "dark-green"} # Default background color.
+'ui.window' = { fg = "mid-green" } # Window border between splits.
+
+'ui.gutter' = { } # Left gutter for diagnostics and breakpoints.
+
+'ui.text' = { fg = "white" } # Default text color.
+'ui.text.focus' = { fg = "white", bg = "mid-green", modifiers = ["bold"] } # Selection highlight in buffer-picker or file-picker.
+'ui.text.info' = { } # Info popup contents (space mode menu).
+
+'ui.cursor' = { fg = "light-blue", modifiers = ["reversed"] } # Fallback cursor colour, non-primary cursors when there are multiple (shift-c).
+'ui.cursor.primary' = { fg = "light-blue", modifiers = ["reversed"] } # The primary cursor when there are multiple (shift-c).
+'ui.cursor.insert' = { fg = "light-blue" } # The cursor in insert mode (i).
+'ui.cursor.select' = { fg = "light-blue" } # The cursor in select mode (v).
+'ui.cursor.match' = { fg = "red", modifiers = ["bold", "reversed"] } # The matching parentheses of that under the cursor.
+
+'ui.selection' = { bg = "mid-green" } # All currently selected text.
+'ui.selection.primary' = { bg = "mid-green" } # The primary selection when there are multiple.
+'ui.cursorline.primary' = { bg = 'mid-green' }
+
+'ui.linenr' = { bg = "dark-green", fg = "gray"  } # Line numbers.
+'ui.linenr.selected' = { bg = "mid-green", fg = "light-green", modifiers = [ "bold" ] } # Current line number.
+
+'ui.virtual' = { fg = "mid-green" } # Namespace for additions to the editing area.
+'ui.virtual.ruler' = { bg = "mid-green" } # Vertical rulers (colored columns in editing area).
+'ui.virtual.whitespace' = { fg = "light-gray"} # Whitespace markers in editing area.
+# NOTE: Indent guides aren't yet added at the time of writing, but will be soon.
+'ui.virtual.indent-guide' = { fg = "light-gray" } # Indentation guides.
+
+'ui.statusline' = { fg = "white", bg = "mid-green" } # Status line.
+'ui.statusline.inactive' = { fg = "white", bg = "dark-green" } # Status line in unfocused windows.
+
+'ui.help' = { bg = "mid-green", fg = "white"} # `:command` descriptions above the command line.
+
+'ui.highlight' = { bg = "mid-green"} # selected contents of symbol pickers (spc-s, spc-S) and current line in buffer picker (spc-b).
+
+'ui.menu' = { fg = "gray", bg = "mid-green" } # Autocomplete menu.
+'ui.menu.selected' = { fg = "white", bg = "autocomp-green" } # Selected autocomplete item.
+
+'ui.popup' = { bg = "mid-green" } # Documentation popups (space-k).
+# 'ui.ppopup.info' = { bg = "midgreen", fg = "gray", modifiers = ["bold"] } # Info popups box (space mode menu).
+
+# SYNTAX HIGHLIGHTING ==============================
+# All the keys here are Treesitter scopes.
+
+'property' = { fg = "red" } # Regex group names.
+'special' = { fg = "mid-blue"} # Special symbols e.g `?` in Rust, `...` in Hare.
+'module' = { } # Unclear, used for Heex module.
```

Comment:
The module key should now be gone: https://github.com/helix-editor/helix/commit/e4e8a39bf79734e70248303782d9b97ef4b3540d

Reply:
changed now!